src/reducers/AddEditSiteReducer.js

```diff
@@ -53,6 +57,38 @@ const INITIAL_STATE = {
     siteDetailCheckboxesKeys: {facilities: [], features: []}
 };
 
+const setInitialEditStateFromSite = ({siteToEdit}) => {
+    const {accessibility, alternateSites = '', approvalState, cellProvider = '', cellStrength = '', coordinate, county = '', description, directions, facilities, features, forest = '', mvum = '', nearestTown, price, siteImageData, title} = siteToEdit;
+    const {latitude, longitude} = coordinate;
+
+    const siteStateToDispatch = {
+        accessibilityOption: accessibility,
+        siteAlternateSitesText: alternateSites,
+        approvalState: approvalState,
+        cellProviderOption: cellProvider,
+        cellStrengthOption: cellStrength,
+        latitudeText: latitude.toString(),
+        readyLatitude: latitude,
+        longitudeText: longitude.toString(),
+        readyLongitude: longitude,
+        countyOption: county,
+        siteDescriptionText: description,
+        siteDirectionsText: directions,
+        siteDetailCheckboxesKeys: {
+            facilities,
+            features
+        },
+        forestOption: forest,
+        mvumOption: mvum,
+        siteNearestTownText: nearestTown,
+        priceOption: price,
+        siteImageData,
+        siteTitleText: title
+    };
+
+    return siteStateToDispatch;
+};
```

Comment:
Helper function to take a `siteToEdit` and grab all of its properties to set on state in the form.